programming_examples/basic/shuffle_transpose/test.cpp

```diff
@@ -0,0 +1,155 @@
+// This file is licensed under the Apache License v2.0 with LLVM Exceptions.
+// See https://llvm.org/LICENSE.txt for license information.
+// SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
+//
+// (c) Copyright 2024 AMD Inc.
+
+#include <cassert>
+#include <cstring>
+#include <fstream>
+#include <iomanip>
+
+#include "xrt/xrt_bo.h"
+#include "xrt/xrt_device.h"
+#include "xrt/xrt_kernel.h"
+
+#ifndef XCLBIN
+#define XCLBIN "build/final.xclbin"
+#endif
+
+#ifndef INSTS_BIN
+#define INSTS_BIN "build/insts.bin"
+#endif
+
+#ifndef KERNEL_NAME
+#define KERNEL_NAME "MLIR_AIE"
+#endif
```

Comment:
Typically we pass these in as arguments, there's a helper function for this too: 

https://github.com/Xilinx/mlir-aie/blob/0d7922324a730d8d9088ac9ccfda868d3c419fd5/runtime_lib/test_lib/test_utils.cpp#L32-L51